### PR TITLE
docs/connection-strings

### PR DESCRIPTION
**Pull request by PaulieScanlon**
- standardize connection strings.

Hi Barry, Hi Daniel, I've tried to be as consistent as possible but there are a few places where we might need to consider some flex on the guidelines.

So... we have two types of connection strings: 

## Square Boys
- `postgres://[user]:[password]...`
## Alex 
- `postgres://alex:AbC123dEf...`

### Square Boys: 
This is the default, but on occasion, there are places where a database has been given a name and is later referred to by name somewhere else on the page. In these scenarios, I've used Square Boys but changed `.../[dbname]` to `.../whatever-the-name-is`
There are and will be more places where `?sslmode=require` needs to be added, I think this looks ok when using Square Boys, e.g. `.../[dbname]?sslmode=require`, but what do you think?

### Alex:
Alex is used where there are places later in the page that explain the parts of a string. Eg.  
- user: alex  
- password: AbC123dEf

In a couple of places I've used Alex because we're modifying the connection string to insert `-pooled` and it looked weird showing this using Square Boys. 

I think I've been consistent with this flexed rule but I could sure use some extra eyes on this. It's an absolute whopper of a PR.

Thanks ✌️

### CODE REVIEW

File: content/docs/connect/connect-pgcli.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ From your terminal or command prompt, run the `pgcli` client with the connection
 <CodeBlock shouldWrap>
 
 ```bash
-postgres://<user>:<password>@ep-raspy-cherry-95040071.us-east-2.aws.neon.tech/<dbname>
+postgres://[user]:[password]@[neon_hostname]/[dbname]
```
Comment (danieltprice) [inline content/docs/connect/connect-pgcli.md:34]:
```suggestion
pgcli postgres://alex:AbC123dEf@ep-cool-darkness-123456.us-east-2.aws.neon.tech/dbname
```

File: content/docs/connect/connect-postgres-gui.md [hunk 2]
```diff
@@ -28,14 +28,14 @@ Neon supports pooled and direct connections to the database. Use a pooled connec
 The connection string includes the role, password, hostname, and database name.
 
 ```text
-postgres://daniel:<password>@ep-raspy-cherry-95040071.us-east-2.aws.neon.tech/neondb
-             ^                                 ^                                ^
-             |- <role>                         |- <hostname>                    |- <database>
+postgres://alex:AbC123dEf@ep-cool-darkness-123456.us-east-2.aws.neon.tech/dbname
+            ^             ^                                               ^
```
Comment (danieltprice) [inline content/docs/connect/connect-postgres-gui.md:32]:
```suggestion
           ^              ^                                               ^
```

File: content/docs/connect/connect-postgres-gui.md [hunk 3]
```diff
@@ -28,14 +28,14 @@ Neon supports pooled and direct connections to the database. Use a pooled connec
...
+            |- <role>     |- <hostname>                                   |- <database>
```
Comment (danieltprice) [inline content/docs/connect/connect-postgres-gui.md:33]:
```suggestion
           |- <role>      |- <hostname>                                   |- <database>
```

File: content/docs/connect/connect-postgres-gui.md [hunk 4]
```diff
@@ -28,14 +28,14 @@ Neon supports pooled and direct connections to the database. Use a pooled connec
...
 ```
 
-- role: `daniel`
-- hostname: `ep-raspy-cherry-95040071.us-east-2.aws.neon.tech`
-- database name: `neondb`
+- role name: `alex`
+- hostname: `ep-cool-darkness-123456.us-east-2.aws.neon.tech
```
Comment (danieltprice) [inline content/docs/connect/connect-postgres-gui.md:37]:
```suggestion
- hostname: `ep-cool-darkness-123456.us-east-2.aws.neon.tech`
```

File: content/docs/connect/connection-errors.md [hunk 5]
```diff
@@ -47,7 +47,7 @@ Neon supports a connection option named `endpoint`, which you can use to identif
 <CodeBlock shouldWrap>
 
 ```txt
-postgres://<user>:<password>@ep-mute-recipe-123456.us-east-2.aws.neon.tech/neondb?options=endpoint%3Dep-mute-recipe-123456
+postgres://[user]:[password]@[neon_hostname]/[dbname]?options=endpoint%3Dep-mute-recipe-123456
```
Comment (danieltprice) [inline content/docs/connect/connection-errors.md:50]:
```suggestion
postgres://[user]:[password]@[neon_hostname]/[dbname]?options=endpoint%3D[endpoint-id]
```

File: content/docs/connect/query-with-psql-editor.md [hunk 6]
```diff
@@ -48,7 +48,7 @@ Neon uses the default Postgres port, `5432`. If you need to specify the port in
 <CodeBlock shouldWrap>
 
 ```bash
-psql postgres://daniel:<password>@ep-raspy-cherry-95040071.us-east-2.aws.neon.tech/neondb
+psql postgres://[user]:[password]@[neon_hostname]/[dbname]
```
Comment (danieltprice) [inline content/docs/connect/query-with-psql-editor.md:51]:
```suggestion
psql postgres://[user]:[password]@[neon_hostname][:port]/[dbname]
```

File: content/docs/guides/branching-pitr.md [hunk 7]
```diff
@@ -59,7 +59,7 @@ To connect to your branch:
    <CodeBlock shouldWrap>
 
    ```bash
-   postgres://daniel:<password>@ep-curly-term-54009904.us-east-2.aws.neon.tech/neondb
+   postgres://[user]:[password]@[neon_hostname]/[dbname]
```
Comment (danieltprice) [inline content/docs/guides/branching-pitr.md:62]:
```suggestion
   psql postgres://[user]:[password]@[neon_hostname]/[dbname]
```

File: content/docs/guides/elixir-ecto.md [hunk 8]
```diff
@@ -37,7 +37,7 @@ You can obtain the connection string for the database from the **Connection Deta
 <CodeBlock shouldWrap>
 
 ```bash
-postgres://<user>:<password>@ep-billowing-sun-767748.us-west-2.aws.neon.tech/friends
+postgres://[user]:[password]@[neon_hostname]/friends
```
Comment (danieltprice) [inline content/docs/guides/elixir-ecto.md:40]:
```suggestion
postgres://alex:AbC123dEf@ep-cool-darkness-123456.us-west-2.aws.neon.tech/friends
```

File: content/docs/guides/exograph.md [hunk 9]
```diff
@@ -31,7 +31,7 @@ You can check the code it created by examining the `src/index.exo` file (which h
 Next, let's set up the Neon database.
 
 ## Create the schema in Neon
-1. Navigate to the Neon Console, select your project, and copy the connection string (which will look something like `postgres://...neon.tech/todo-db`).
+1. Navigate to the Neon Console, select your project, and copy the connection string (which will look something like `postgres://alex:AbC123dEf@ep-cool-darkness-123456.us-east-2.aws.neon.tech/dbname`).
```
Comment (danieltprice) [inline content/docs/guides/exograph.md:34]:
```suggestion
1. Navigate to the Neon Console, select your project, and copy the connection string, which will look something like this: `postgres://alex:AbC123dEf@ep-cool-darkness-123456.us-east-2.aws.neon.tech/dbname`.
```

File: content/docs/guides/hasura.md [hunk 10]
```diff
@@ -41,7 +41,7 @@ Use the following instructions to connect to an existing Neon database from Hasu
 - A connection string for a branch in Neon project that includes your password. For example:
```
Comment (danieltprice) [inline content/docs/guides/hasura.md:41]:
```suggestion
- A connection string for a branch in your Neon project:
```

File: content/docs/guides/node.md [hunk 11]
```diff
@@ -59,7 +59,7 @@ Add a `.env` file to your project directory and add your Neon connection string
 <CodeBlock shouldWrap>
 
 ```shell
-DATABASE_URL=postgres://<user>:<password>@ep-snowy-unit-550577.us-east-2.aws.neon.tech/neondb?options=endpoint%3Dep-snowy-unit-550577
+DATABASE_URL=ppostgres://[user]:[password]@[neon_hostname]/[dbname]
```
Comment (danieltprice) [inline content/docs/guides/node.md:62]:
```suggestion
DATABASE_URL=postgres://[user]:[password]@[neon_hostname]/[dbname]
```

File: content/docs/guides/read-replica-data-analysis.md [hunk 12]
```diff
@@ -97,10 +97,10 @@ Connecting to a read replica is the same as connecting to any branch, except you
...
     The information in your connection string corresponds to the following connection details:
 
-    - role: `daniel`
-    - password:`<pasword>`
-    - hostname: `ep-mute-rain-952417.us-east-2.aws.neon.tech`
-    - database name: `neondb`. This is the ready-to-use database created with your Neon project. Your database name may differ.
+    - role: `alex`
+    - password:`AbC123dEf`
+    - hostname: `ep-cool-darkness-123456.us-east-2.aws.neon.tech`
+    - database name: `dbname`. This is the ready-to-use database created with your Neon project. Your database name may differ.
```
Comment (danieltprice) [inline content/docs/guides/read-replica-data-analysis.md:103]:
```suggestion
    - database name: `dbname`. Your database name may differ.
```

File: content/docs/guides/read-replica-prisma.md [hunk 13]
```diff
@@ -83,8 +83,8 @@ Connecting to a read replica is the same as connecting to any branch in a Neon p
 In your `.env` file, set a `DATABASE_REPLICA_URL` environment variable to the connection string of your read replica. Your `.env` file should look something like this, with your regular `DATABASE_URL` and the newly added `DATABASE_REPLICA_URL`.
 
 ```text
-DATABASE_URL="postgres://daniel:<password>@ep-raspy-cherry-95040071.us-east-2.aws.neon.tech/neondb"
-DATABASE_REPLICA_URL="postgres://daniel:<password>@ep-damp-cell-18160816.us-east-2.aws.neon.tech/neondb"
+DATABASE_URL="postgres://alex:AbC123dEf@ep-cool-darkness-123456.us-east-2.aws.neon.tech/dbname"
+DATABASE_REPLICA_URL="postgres://[user]:[password]@[neon_hostname]/[dbname]"
```
Comment (danieltprice) [inline content/docs/guides/read-replica-prisma.md:87]:
```suggestion
DATABASE_REPLICA_URL="postgres://alex:AbC123dEf@ep-damp-cell-123456.us-east-2.aws.neon.tech/[dbname]"
```

File: content/docs/guides/read-replica-prisma.md [hunk 14]
```diff
@@ -83,8 +83,8 @@ Connecting to a read replica is the same as connecting to any branch in a Neon p
...
 ```
 
 Notice that the `endpoint_id` for the read replica compute differs. The read replica is a different compute instance and therefore has a different `endpoint_id`.
```
Comment (danieltprice) [inline content/docs/guides/read-replica-prisma.md:90]:
```suggestion
Notice that the `endpoint_id` (`ep-damp-cell-123456`) for the read replica compute differs. The read replica is a different compute instance and therefore has a different `endpoint_id`.
```

File: content/docs/guides/stepzen.md [hunk 15]
```diff
@@ -33,7 +33,7 @@ You can seed the database directly from the terminal by running the following `p
 <CodeBlock shouldWrap>
 
 ```bash
-psql postgres://daniel:<password>@ep-raspy-cherry-95040071.us-east-2.aws.neon.tech/neondb < init.sql
+psql postgres://alex:AbC123dEf@ep-cool-darkness-123456.us-east-2.aws.neon.tech/dbname < init.sql
```
Comment (danieltprice) [inline content/docs/guides/stepzen.md:36]:
```suggestion
psql postgres://[user]:[password]@[neon_hostname]/[dbname] < init.sql
```

File: content/docs/guides/vercel-manual.md [hunk 16]
```diff
@@ -26,30 +26,30 @@ You can these details from the **Connection Details** widget on the **Neon Dashb
 The connection string includes the role name, hostname, and database name. For example:
 
 ```text
-postgres://daniel:<password>@ep-raspy-cherry-95040071.us-east-2.aws.neon.tech/neondb
-             ^                                 ^                           ^
-             |- <role>                         |- <hostname>               |- <database>
+postgres://alex:AbC123dEf@ep-cool-darkness-123456.us-east-2.aws.neon.tech/dbname
+            ^             ^                                               ^
```
Comment (danieltprice) [inline content/docs/guides/vercel-manual.md:30]:
```suggestion
           ^              ^                                               ^
```

File: content/docs/guides/vercel-manual.md [hunk 17]
```diff
@@ -26,30 +26,30 @@ You can these details from the **Connection Details** widget on the **Neon Dashb
...
+            |- <role>     |- <hostname>                                   |- <database>
```
Comment (danieltprice) [inline content/docs/guides/vercel-manual.md:31]:
```suggestion
           |- <role>      |- <hostname>                                   |- <database>
```

File: content/docs/import/migrate-aws-dms.md [hunk 18]
```diff
@@ -29,18 +29,18 @@ Complete the following steps before you begin:
 4. Select **Target endpoint** as the **Endpoint type**.
 5. Provide an **Endpoint identifier** label for your new target endpoint. In this guide, we use `neon` as the identifier.
 6. In the **Target engine** drop-down menu, select `PostgreSQL`.
-7. Under **Access to endpoint database**, select **Provide access information manually** and enter the information outlined below. You can obtain the connection details from your Neon connection string, which you can find in the **Connection Details** widget on the Neon **Dashboard**. Your  connection string will look similar to this: `postgres://daniel:<password>@ep-curly-term-54009904.us-east-2.aws.neon.tech/neondb`.
+7. Under **Access to endpoint database**, select **Provide access information manually** and enter the information outlined below. You can obtain the connection details from your Neon connection string, which you can find in the **Connection Details** widget on the Neon **Dashboard**. Your  connection string will look similar to this: `postgres://alex:AbC123dEf@ep-cool-darkness-123456.us-east-2.aws.neon.tech/dbname"`.
```
Comment (danieltprice) [inline content/docs/import/migrate-aws-dms.md:32]:
```suggestion
7. Under **Access to endpoint database**, select **Provide access information manually** and enter the information outlined below. You can obtain the connection details from your Neon connection string, which you can find in the **Connection Details** widget on the Neon **Dashboard**. Your  connection string will look similar to this: `postgres://alex:AbC123dEf@ep-cool-darkness-123456.us-east-2.aws.neon.tech/neondb"`.
```

File: content/docs/manage/projects.md [hunk 19]
```diff
@@ -13,7 +13,7 @@ A Neon project is created with the following resources, by default:
 
 - A primary branch called `main`. You can create child branches from the primary branch or from a previously created branch. For more information, see [Manage branches](/docs/manage/branches).
 - A single read-write compute endpoint, which is the compute instance associated with a branch. For more information, see [Manage computes](/docs/manage/endpoints).
-- A ready-to-use database, called `neondb`, which resides in the project's primary branch.
+- A ready-to-use database, called `dbname`, which resides in the project's primary branch.
```
Comment (danieltprice) [inline content/docs/manage/projects.md:16]:
```suggestion
- A ready-to-use database, called `neondb`, which resides in the project's primary branch.
```

File: content/docs/manage/projects.md [hunk 20]
```diff
@@ -31,7 +31,7 @@ To create a Neon project:
   
 4. Click **Create Project**.
 
-Upon creating a project, you are presented with a dialog that provides your connection details for a ready-to-use `neondb` database. The connection details include your password.
+Upon creating a project, you are presented with a dialog that provides your connection details for a ready-to-use `dbname` database. The connection details include your password.
```
Comment (danieltprice) [inline content/docs/manage/projects.md:34]:
```suggestion
Upon creating a project, you are presented with a dialog that provides your connection details for a ready-to-use `neondb` database. The connection details include your password.
```

File: content/docs/manage/projects.md [hunk 21]
```diff
@@ -148,7 +148,7 @@ curl 'https://console.neon.tech/api/v2/projects' \
 }' | jq
 ```
 
-The response includes information about the roles, the ready-to-use database (`neondb)`), the primary branch (`main`), and the read-write compute endpoint that is created with the project.
+The response includes information about the roles, the ready-to-use database (`dbname)`), the primary branch (`main`), and the read-write compute endpoint that is created with the project.
```
Comment (danieltprice) [inline content/docs/manage/projects.md:151]:
```suggestion
The response includes information about the roles, the ready-to-use database (`neondb`), the primary branch (`main`), and the read-write compute endpoint that is created with the project.
```

File: content/docs/reference/glossary.md [hunk 22]
```diff
@@ -143,7 +143,7 @@ A billing metric that measures the total volume of data transferred out of Neon
 
 ### Database
 
-A named collection of database objects. A Neon project has a ready-to-use database named `neondb` which resides in the default `public` schema. A Neon project can contain multiple databases. Users cannot manipulate system databases, such as the `postgres`, `template0`, or `template1` databases.
+A named collection of database objects. A Neon project has a ready-to-use database named `dbname` which resides in the default `public` schema. A Neon project can contain multiple databases. Users cannot manipulate system databases, such as the `postgres`, `template0`, or `template1` databases.
```
Comment (danieltprice) [inline content/docs/reference/glossary.md:146]:
```suggestion
A named collection of database objects. A Neon project has a ready-to-use database named `neondb` which resides in the default `public` schema. A Neon project can contain multiple databases. Users cannot manipulate system databases, such as the `postgres`, `template0`, or `template1` databases.
```

File: content/release-notes/2023-05-16-storage-and-compute.md [hunk 23]
```diff
@@ -7,7 +7,7 @@
    <CodeBlock shouldWrap>
 
   ```txt
-  postgres://<user>:<password>@ep-mute-recipe-239816.us-east-2.aws.neon.tech/main?options=endpoint%3Dep-mute-recipe-239816
+  postgres://[user]:[password]@[neon_hostname]/[dbname]?options=endpoint%3Dep-mute-recipe-239816
```
Comment (danieltprice) [inline content/release-notes/2023-05-16-storage-and-compute.md:10]:
```suggestion
  postgres://[user]:[password]@[neon_hostname]/[dbname]?options=endpoint%3D[endpoint_id]
```

File: src/components/pages/aws-israel/cli/cli.jsx [hunk 24]
```diff
@@ -4,7 +4,7 @@ import GradientLabel from 'components/shared/gradient-label';
 
 const code = `  > neonctl connection-string
 
-  postgres://<user>:<password>@epl-792025.us-east-2.aws.neon.tech/neondb
+  postgres://[user]:[password]@[neon_hostname]/[dbname]
```
Comment (danieltprice) [inline src/components/pages/aws-israel/cli/cli.jsx:7]:
```suggestion
  postgres://[user]:[password]@ep-cool-darkness-123456.il-central-1.aws.neon.tech/neondb
```

**Review by danieltprice — APPROVED** [review]
Thanks for this, Paul. I have made a couple of commits for things that fell outside of the modified lines. For the modified lines, I left suggestions for you to commit.

I'll follow up with some screen capture updates, which I'll do once this PR merged.

**Comment by PaulieScanlon** [comment]
@danieltprice. Thanks! Sorry, this one was so big and thanks for taking the time to go through it. The next few PRs (hopefully) will be smaller.

**Pull request merged**